internal/configs/testdata/tofu-and-tf-files/variables.tofu

```diff
@@ -0,0 +1,3 @@
+variable "should_override" {
+    default = "not overridden"
+}
```

Comment:
Data here is exactly the same as with the `variables.tf`, so if the test passes we can't really tell if `variables.tofu` was used or whether `variables.tf` was used

Same for `variables.tofu.json`, `variable_override.tf` and `variable_override.tf.json`

Reply:
I removed `variables.tofu`. I am checking that the correct override file is being picked in the test (the .tofu overrides have a different string than the .tf files).